index.yaml

```diff
@@ -25,6 +25,65 @@ distributions:
       #centos: eloquent/release-centos-build.yaml
     source_builds:
       default: eloquent/source-build.yaml
+  foxy:
+    ci_builds:
+      # benchmark: foxy/ci-benchmark.yaml
+      # nightly-connext: foxy/ci-nightly-connext.yaml
+      # nightly-cross-vendor-connext-cyclonedds: foxy/ci-nightly-cross-vendor-connext-cyclonedds.yaml
+      # nightly-cross-vendor-connext-fastrtps: foxy/ci-nightly-cross-vendor-connext-fastrtps.yaml
+      # nightly-cross-vendor-connext-fastrtps-dynamic: foxy/ci-nightly-cross-vendor-connext-fastrtps-dynamic.yaml
+      # nightly-cross-vendor-cyclonedds-fastrtps: foxy/ci-nightly-cross-vendor-cyclonedds-fastrtps.yaml
+      # nightly-cross-vendor-cyclonedds-fastrtps-dynamic: foxy/ci-nightly-cross-vendor-cyclonedds-fastrtps-dynamic.yaml
+      # nightly-cross-vendor-fastrtps-fastrtps-dynamic: foxy/ci-nightly-cross-vendor-fastrtps-fastrtps-dynamic.yaml
+      # nightly-cyclonedds: foxy/ci-nightly-cyclonedds.yaml
+      # nightly-debug: foxy/ci-nightly-debug.yaml
+      # nightly-extra-rmw-release: foxy/ci-nightly-extra-rmw-release.yaml
+      # nightly-fastrtps: foxy/ci-nightly-fastrtps.yaml
+      # nightly-fastrtps-dynamic: foxy/ci-nightly-fastrtps-dynamic.yaml
+      # nightly-performance: foxy/ci-nightly-performance.yaml
+      nightly-release: foxy/ci-nightly-release.yaml
+      overlay: foxy/ci-overlay.yaml
+    doc_builds: {}
+    notification_emails:
+      - steven+build.test.ros2.org@openrobotics.org
+      - scott+build.test.ros2.org@openrobotics.org
+      - william+build.test.ros2.org@openrobotics.org
```

Comment:
I added myself to foxy and rolling, but I also added you guys @nuclearsandwich @cottsay, but I can change that if you prefer.

Reply:
This is mostly a safety catch in case emails magically work but smtp is disabled on the test host